.yarn/releases/yarn-1.22.15.cjs

```diff
@@ -98,192 +101,192 @@ module.exports = require("path");
 /* unused harmony export __makeTemplateObject */
 /* unused harmony export __importStar */
 /* unused harmony export __importDefault */
-/*! *****************************************************************************

```

Comment:
What happened here? bulk newline change or something?

Reply:
Autolint.  Thanks for pointing it out... I've reverted it.